src/Features/CSharpTest/ConvertLocalFunctionToMethod/ConvertLocalFunctionToMethodTests.cs

```diff
@@ -1115,4 +1115,51 @@ struct S
             }
             """);
     }
+
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/64904")]
+    public async Task TestNameofReferenceInParameterInitializer()
+    {
+        await TestInRegularAndScript1Async(
+            """
+            using System;
+
+            sealed class A : Attribute
+            {
+                public A(string _) { }
+                public string P { get; set; }
+            }
+
+            class C
+            {
+                void M()
+                {
+                    void [||]M2([A(nameof(b), P = nameof(b))] string b)
```

Comment:
>A(nameof(b), P = nameof(b [](http://example.com/codeflow?start=33&length=25)

would it be worth testing that it doesn't use nameof for something for P in the following?

void M2([A(nameof(b), P = "b")] string b)

Reply:
nah :)

and basically i dont' care.  the chance of the user writing that seems miniscule.